pubsub/cloud-client/src/main/java/com/example/pubsub/QuickstartSample.java

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright 2016, Google, Inc.
+  Copyright 2017, Google, Inc.
```

Comment:
It turns out that the copyright year should actually be the [date of first publish](http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated), and establishes how far back the copyright extends; so it's generally more advantageous for it to be lower.